game-app/game-core/src/main/java/games/strategy/triplea/odds/calculator/BattleCalculatorPanel.java

```diff
@@ -633,12 +634,12 @@ private void updateStats() {
     final double avgDefIfDefWon = results.getAverageDefendingUnitsLeftWhenDefenderWon();
     defenderLeftWhenDefenderWon.setText(
         Double.isNaN(avgDefIfDefWon)
-            ? "N/A"
+            ? DownloadMapsWindow.TEXT_ABBREVIATION_NOT_APPLICABLE
```

Comment:
IMO, leave the string inlined here. Why should `DownloadMapsWindow` own the `N/A` string?

I don't think the constant is really helping very much, it's okay to let things be inlined and duplicated sometimes.

Reply:
Sure, but we're not making it translatable just yet. Let's do this work when we do the other thing. Otherwise this is just half done. Let's leave it be unless we do the whole effort.

Reply:
We are outsourcing the text and combine 3 places into one. I do not see what benefit it has to undo it in order to redo it again in the future.

Reply:
> I do not see what benefit it has to undo it in order to redo it again in the future.

I'm suggesting that is premature optimization. Who is to say this class will even exist in its current form when we do redo it. The rule of thumb, have the simplest possible code for now, do the other stuff when it is needed. Would you prefer to add a feature to the simplest possible code base, or would you prefer to add a feature that tried to predict all the features that were going to be added?

To another extent, this PR is to "calculate the size of downloaded maps when selected." What does that have to do with internationalizing the string N/A?